src/plugins/LADTable/components/LADRow.vue

```diff
@@ -89,43 +94,80 @@ export default {
             .telemetry
             .subscribe(this.domainObject, this.updateValues);
 
-        this.openmct
-            .telemetry
-            .request(this.domainObject, {strategy: 'latest'})
-            .then((array) => this.updateValues(array[array.length - 1]));
+        this.requestHistory();
     },
     destroyed() {
         this.stopWatchingMutation();
         this.unsubscribe();
-        this.openmct.off('timeSystem', this.updateTimeSystem);
+        this.openmct.time.off('timeSystem', this.updateTimeSystem);
+        this.openmct.time.off('bounds', this.updateBounds);
     },
     methods: {
         updateValues(datum) {
-            this.timestamp = this.formats[this.timestampKey].format(datum);
-            this.value = this.formats[this.valueKey].format(datum);
-
-            var limit = this.limitEvaluator.evaluate(datum, this.valueMetadata);
+            let newTimestamp = this.formats[this.timestampKey].parse(datum),
+                shouldUpdate = false,
+                limit;
+
+            if(this.inBounds(newTimestamp)) {
```

Comment:
I think the logic can be simplified, and moved into a function to clean things up a bit here.  eg.
``` JavaScript
function shouldUpdate() {
     return (timestamp === undefined) ||
        (this.inBounds(newTimestamp) && 
        newTimestamp > this.timestamp);
}
```
Function could then be called from an if statement in `updateValues()`

Reply:
Agreed, Shefali had mentioned a similar solution in the lad-testing PR. I've moved the logic to a separate function that does the check.